tests/python/test_atari_env.py

```diff
@@ -34,30 +41,33 @@ def test_roms_register():
 
 
 @pytest.mark.parametrize(
-    "env_id",
-    [
-        env_id
-        for env_id, spec in gymnasium.registry.items()
-        if spec.entry_point == "ale_py.env:AtariEnv"
-    ],
+    "env_id,continuous",
```

Comment:
For the future this can be two separate @pytest.mark.parameterize, the first for the env id and the second continuous